### DISCUSSION TITLE
React Router v6.x

### DISCUSSION THREAD

**Discussion by nerdess** [dasniko/keycloak-reactjs-demo#36] [Q&A] [answered]
Does anyone have a fork of this code using React Router v6.x?

**Comment by AlmondMilk4242** [comment]
any updates on this would be really appreceated

**Comment by dasniko** [comment, marked as answer]
see https://github.com/dasniko/keycloak-reactjs-demo/commit/5846674e6af3c60487117e2ba6debcfe9e66b792
